### DISCUSSION TITLE
Customer Facing Mobile app integration

### DISCUSSION THREAD

**Discussion by devrulabs** [chatwoot/chatwoot#2115] [Q&A]
hello, we would like to implement the ability for our customers to be able to initiate a chat from our Mobile App. is there an SDK to build chat feature into the Mobile App or should we use the API channel to do this ?

PS : This is not the Mobile App the Agents will use, this is the Mobile App the customers will be using to initiate the chat.

**Comment by sojan-official** [comment]
https://github.com/chatwoot/chatwoot-react-native-widget

**Reply by devrulabs** [reply]
Thanks so much for the quick response :)
